server/src/main/java/com/cloud/ha/HighAvailabilityManagerImpl.java

```diff
@@ -844,29 +851,29 @@ public boolean configure(final String name, final Map<String, Object> xmlParams)
         value = params.get("force.ha");
         _forceHA = Boolean.parseBoolean(value);
 
-        value = params.get("time.to.sleep");
-        _timeToSleep = (long)NumbersUtil.parseInt(value, 60) * 1000;
+        value = params.get(TimeToSleep.key());
+        _timeToSleep = NumbersUtil.parseLong(value, TimeToSleep.value()) * MILLISECONDS_TO_SECONDS_FACTOR;
```

Comment:
I think what @GabrielBrascher suggested is
```
_timeToSleep = TimeToSleep.value() * MILLISECONDS_TO_SECONDS_FACTOR;
```

Reply:
It is already a DB fetch made, where the values are read. The change may result in 4 additional queries.